models/recipe.model.js

```diff
@@ -0,0 +1,81 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const { User, UserSchema } = require("./user.model");
+
+const TagSchema = new Schema({
+    id: {
+        type: Number,
+        required: true,
+    },
+    name: {
+        type: String,
+        required: true,
+    },
+});
+
+module.exports = TagSchema;
+module.exports = Tag = mongoose.model("tag", TagSchema);
+
+const IngredientSchema = new Schema({
+    id: {
+        type: Number,
+        required: true,
+    },
+    name: {
+        type: String,
+        required: true,
+    },
+    amount: {
+        type: Number,
+        required: true,
+    },
+    unit: {
```

Comment:
Should we consider making this an enum? That may be too much effort but we could include conversions in our app yk?

Reply:
I guess that will just break the app though if a recipe includes an unrecognized unit of measurement